core/field_dropdown.ts

```diff
@@ -633,7 +633,13 @@ export class FieldDropdown extends Field<string> {
   /**
    * Use the `getText_` developer hook to override the field's text
    * representation.  Get the selected option text.  If the selected option is
-   * an image we return the image alt text.
+   * an image we return the image alt text. If the selected option is
+   * an HTMLElement, return the title, ariaLabel, or innerText of the
+   * element.
+   *
+   * If you use HTMLElement options in node and call this function,
```

Comment:
```suggestion
   * If you use HTMLElement options in Node.js and call this function,
```

Just to avoid potential ambiguity in language with other 'node' things (of which there are certainly plenty).